styleguide/source/_patterns/01-molecules/06-components/15-table/table.twig

```diff
@@ -0,0 +1,50 @@
+<div class="table">
+
+  {% include 'atoms-h5' with { 'content': 'Lorem Ipsum Dolor Amet Consectetur Adipiscing', 'class': 'table-header_title' } %}
+
+  <table>
```

Comment:
Do we want to add a class here of table instead of applying styling directly to html element?